lib/api/accounts.js

```diff
@@ -89,15 +135,15 @@ module.exports = function (app) {
 				return cb(null, account);
 			}
 			return request.get({
-				url: `${app.get('lisk address')}/api/accounts/delegates/?address=${account.address}`,
+				url: `${app.get('lisk address')}/accounts?address=${account.address}`,
 				json: true,
 			}, (err, response, body) => {
 				if (err || response.statusCode !== 200) {
 					return cb(err || 'Response was unsuccessful');
-				} else if (body.success === true) {
-					account.votes = (body.delegates !== undefined &&
-						body.delegates !== null && body.delegates.length > 0) ?
-						body.delegates : null;
+				} else if (Array.isArray(body.data)) {
+					account.votes = (body.data !== undefined &&
```

Comment:
Because of test above (`Array.isArray(body.data)`) - `body.data` cannot be `undefined`/`null`, so those checks are redundant.